packages/experience/src/pages/ForgotPassword/index.tsx

```diff
@@ -15,9 +13,9 @@ import ForgotPasswordForm from './ForgotPasswordForm';
 
 const ForgotPassword = () => {
   const { isForgotPasswordEnabled, enabledMethodSet } = useForgotPasswordSettings();
-  const { state } = useLocation();
   const { t } = useTranslation();
   const enabledMethods = [...enabledMethodSet];
+  const { identifierInputValue } = useContext(UserInteractionContext);
```

Comment:
There are multiple places where the `location.state` is used to carry over identifiers across pages, e.g code verification page will read the value from the state to display the current identifier value. 
Better update all those pages as well?

Reply:
Good call, updated, now the identifier value will be read from `UserInteractionContext`.